Clients/GUI/Pages/SignIn.razor

```diff
@@ -35,21 +32,23 @@
     bool isHidden = true;
     string errorMessage = "";
 
-    private void TrySignIn()
+    private async Task TrySignIn()
```

Comment:
why return's type is task and not void? Who use this task in main thread? Are you sure we need mark this method async? We can use getresult method instead await. I think, we musd  do it, because user can send only one pacage of data per time

Reply:
> why return's type is task and not void? Who use this task in main thread? Are you sure we need mark this method async? We can use getresult method instead await. I think, we musd do it, because user can send only one pacage of data per time

Check updated version of SignInner